pkg/util/constant.go

```diff
@@ -33,10 +33,12 @@ const (
 	JivaCasType = "jiva"
 	// LVMCasType cas type name
 	LVMCasType = "localpv-lvm"
-	// LocalHostpathCasType cas type name
-	LocalHostpathCasType = "localpv-hostpath"
+	// LocalPvHostpathCasType cas type name
+	LocalPvHostpathCasType = "localpv-hostpath"
 	// LocalDeviceCasType cas type name
 	LocalDeviceCasType = "localpv-device"
+	// Local-hostpath cas type name
+	LocalHostpathCasType = "local-hostpath"
```

Comment:
```suggestion
```

This can be removed, I've discussed this with Abhinandan, we are sort of in the fence when it comes to naming things right now, but we are positive that removing this will not change anything.